source/administration/mmctl-cli-tool.rst

```diff
@@ -8,34 +8,22 @@ Being installed locally allows a System Admin to run CLI commands even in instan
 server (e.g., via SSH). This tool is currently in beta and can be used alongside the Mattermost CLI tool.
 In the future, the Mattermost CLI tool will be deprecated.
 
-This feature was developed to a large extent by community contributions and we'd like to extend our gratitude to the contributors who have
-worked on this project. We are currently accepting pull requests for Help Wanted issues
+This feature was developed to a large extent by community contributions and we'd like to extend our gratitude to the contributors who have worked on this project. We are currently accepting pull requests for Help Wanted issues
 in the `mattermost-server <https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+mmctl>`__ repo. You can learn more about
```

Comment:
Not strictly related: this link uses the `Help Wanted` and `Area/mmctl` GitHub labels:

```suggestion
in the `mattermost-server <https://github.com/mattermost/mattermost-server/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22+label%3AArea%2Fmmctl>`__ repo. You can learn more about
```